src/components/input-field/input-field.tsx

```diff
@@ -112,6 +112,7 @@ export class InputField {
     private action: EventEmitter<void>;
 
     constructor() {
+        this.renderHelperText = this.renderHelperText.bind(this);
```

Comment:
There is no need to bind this function, since it is called with `this.`

Reply:
⚡️